templates/data/summary.html

```diff
@@ -11,7 +11,14 @@
 
 {%- block content -%}
   <h3>Details for track {{ mbid }}</h3>
-  {% if mbid %}
+
+  {%- if not metadata -%}
+    <div class="alert alert-warning" role="alert">
```

Comment:
This isn't really a warning. It's not actionable by the person viewing the data.
Maybe instead we can have a "Source" label in the metadata table (where it currently says 'value')